_rules/SC2-1-2-no-keyboard-trap-non-standard-navigation.md

```diff
@@ -29,6 +29,8 @@ authors:
 
 The rule applies to any HTML or SVG element on a web page that is [focusable][] where focus cannot cycle to the browser UI by using [standard keyboard navigation][].
 
+**Note**: The WCAG 2.0 success criterion 2.1.2 applies to all content where focus can be moved to through keyboard navigation.
+
 ### Expectation 1
 
 For each target element help information is [visible on the page][] and [exposed to assistive technologies][] or can be accessed from within the keyboard trap.
```

Comment:
This "or can be accessed" part is a little clunky. Can this be rephrased without changing its meaning?